br/use-br-command-line-tool.md

```diff
@@ -481,9 +506,25 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 > - the system variable tables (`mysql.tidb`，`mysql.global_variables`) cannot be restored.
 > - the user information tables (such as `mysql.user` and `mysql.columns_priv`) cannot be restored.
 > - GC data cannot be restored.
-> 
+>
 > Restoring system tables might cause more compatibility issues. To avoid unexpected issues, **DO NOT** restore system tables in the production environment.
 
+### Decrypt backup data
```

Comment:
what about the title of "Encrypt data during backup" and "Decrypt data during restore" int below context ?